executor/src/witgen/jit/test_util.rs

```diff
@@ -1,42 +1 @@
-use itertools::Itertools;
```

Comment:
Yes, this module is empty now. But #2276 adds to it, so if that gets merge first, it won't be empty on `main`.

Reply:
weird